tests/unit/core/i18n/test_utils.js

```diff
@@ -487,5 +479,12 @@ describe('i18n utils', () => {
       sinon.assert.calledWith(toLocaleStringSpy, 'fr');
       sinon.assert.notCalled(numberFormatSpy);
     });
+
+    it('always returns a scoped moment instance', () => {
+      const i18n = utils.makeI18n({}, 'fr', FakeJed);
+      // We modify the moment locale, globally.
```

Comment:
This comment threw me off when I read the test because I wasn't sure why you were modifying the global instance. This comment would have helped me understand why:

```
// Modifying the locale globally does not affect the instance.
```

Reply:
ah sorry for that.